app/presenters/hyku/menu_presenter.rb

```diff
@@ -50,16 +50,16 @@ def show_task?
     end
 
     # Draw a collaspable menu section. The passed block should contain <li> items.
-    # Override Hyrax 3.5.0 to pass in html_options
+    # Override Hyrax v5.0.0rc2 to pass in title attribute
     # rubocop:disable Metrics/ParameterLists
-    def collapsable_section(text, id:, icon_class:, open:, **html_options, &block)
```

Comment:
This breaks the outward interface; namely that we're capturing all additional key value pairs as `**html_options`; consider instead adding `:title` as a keyword and passing that along.